hasher-matcher-actioner/terraform/authentication/main.tf

```diff
@@ -73,3 +73,83 @@ resource "aws_cognito_user_pool_client" "webapp_and_api_user_pool_client" {
   access_token_validity  = 60
   id_token_validity      = 60
 }
+
+## Integrations Lambda - maybe move to its own module
```

Comment:
As we discussed at standup, my belief is that partners will already have some event processor which would then submit to HMA. 

This appears too specific to be generalizable (even if made optional) for multiple partners. Vote for not adding integration lambdas to the main terraform setup. 

If partners want to receive other payloads from their own systems, my suggestion would be for them to expand on the submit APIs in submit.py. Since we intend on providing support, we could nudge them towards that.